Source/WebCore/Modules/webxr/WebXROpaqueFramebufferCocoa.cpp

```diff
@@ -162,27 +161,15 @@ void WebXROpaqueFramebuffer::startFrame(const PlatformXR::FrameData::LayerData&
         m_completionSyncEvent = MachSendRight(data.layerSetup->completionSyncEvent);
     }
 
+    gl->bindFramebuffer(GL::FRAMEBUFFER, m_drawFramebuffer->object());
```

Comment:
The foveation setup is very buggy, if it still needs the correct framebuffer..
if you move the bind here, the setupFramebuffer might not have a expected framebuffer bound.